src/application/service/tests/AddResultsToTest.ts

```diff
@@ -63,4 +74,20 @@ export class AddResultsToTest {
     }
     return ConfidenceLevel.LOW;
   }
+
+  private async sendEmailForTestResult(patient: User) {
+    const emailTemplate = fs.readFileSync(__dirname + '/../../../../assets/emails/new-test-result.html').toString();
```

Comment:
Let's do this async, using `fs.readFile`. Node is single-threaded, so using any blocking I/O API-s like this one results in the thread not serving any requests while it's waiting for this file.

Reply:
👍